x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/with_security_context.tsx

```diff
@@ -51,8 +53,15 @@ export const withSecurityContext = <P extends {}>({
       store = createStore(
         combineReducers({
           management: managementReducer,
+          app: appReducer,
         }),
-        { management: undefined },
+        {
+          management: undefined,
+          // @ts-ignore ignore this error as we just need the enableExperimental and it's temporary
+          app: {
+            enableExperimental: allowedExperimentalValues,
```

Comment:
This is ok for now, but we might be able to get the "real" values here by storing them in a singleton service inside of this `ingest_manager_integration` directory (a service similar to our `KibanaServices` in security solution ([here](https://github.com/elastic/kibana/blob/4584a8b570402aa07832cf3e5b520e5d2cfa7166/x-pack/plugins/security_solution/public/common/lib/kibana/services.ts#L13)).

Reply:
Finally I did that because was needed for the FTR test https://github.com/elastic/kibana/pull/111708/commits/a13df35e054e963f96625cf14de5e683cd80e623